app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/EditGraphicNotesFragment.kt

```diff
@@ -0,0 +1,110 @@
+package dev.arkbuilders.arkmemo.ui.fragments
+
+import android.os.Build
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import by.kirich1409.viewbindingdelegate.viewBinding
+import dagger.hilt.android.AndroidEntryPoint
+import dev.arkbuilders.arkmemo.R
+import dev.arkbuilders.arkmemo.databinding.FragmentEditNotesBinding
+import dev.arkbuilders.arkmemo.models.DEFAULT_TITLE
+import dev.arkbuilders.arkmemo.models.GraphicNote
+import dev.arkbuilders.arkmemo.ui.activities.MainActivity
+import dev.arkbuilders.arkmemo.ui.viewmodels.GraphicNotesViewModel
+import dev.arkbuilders.arkmemo.ui.viewmodels.NotesViewModel
+import dev.arkbuilders.arkmemo.utils.observeSaveResult
+
+@AndroidEntryPoint
+class EditGraphicNotesFragment: Fragment(R.layout.fragment_edit_notes) {
+
+    private val activity by lazy {
+        requireActivity() as MainActivity
+    }
+
+    private val binding by viewBinding(FragmentEditNotesBinding::bind)
+
+    private val graphicNotesViewModel: GraphicNotesViewModel by viewModels()
+    private val notesViewModel: NotesViewModel by activityViewModels()
+
+    private var note = GraphicNote( title = "")
```

Comment:
Empty title initialization doesn't seem to be needed here because the title is empty by default.